recipe/meta.yaml

```diff
@@ -43,64 +43,111 @@ requirements:
     - flask_cors
     - jsondiff >=1.1.2
     - aws-xray-sdk !=0.96,>=0.93
-    - sshpubkeys >=3.1.0,<4.0
+    - sshpubkeys >=3.1.0
     - idna >=2.5,<4
+    - graphql-core
+    - setuptools
+    - importlib_metadata
```

Comment:
According to their setup.py they only want importlib_metadata for py <3.8 (and select stdlib version otherwise). I think making this conditional makes the package no longer noarch though. Not sure what the best practice is here.

Reply:
@mariusvniekerk any thoughts on best way to approach this?

Reply:
Given they are using it (try stdlib first, fall back to package) https://github.com/spulec/moto/blob/01ae8c01a63b8a5b24e1f953b35c6d9a06e56cbf/moto/core/custom_responses_mock.py#L10-L13

I think it is safe to have this present for all versions and just stick with noarch.